admin/app/components/solidus_admin/properties/edit/component.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Properties::Edit::Component < SolidusAdmin::BaseComponent
+    def initialize(page:, property:)
+      @page = page
+      @property = property
+    end
+
+    def form_id
+      dom_id(@property, "#{stimulus_id}_edit_property_form")
+    end
+  end
```

Comment:
This file is mostly misindented.

Reply:
@jarednorman thank you for your message I have just already made rebase in this PR and follow the solidus guideline let me know if need a cleaner PR I can upload 1 single commit with all changes ... let me know please